tests/Feature/Booking/BookingUpdateTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Tests\Feature\Booking;
+
+use App\Modules\Bookings\Booking;
+use App\Position;
+use Carbon\Carbon;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Nuwave\Lighthouse\Testing\MakesGraphQLRequests;
+use Tests\TestCase;
+use Tests\Unit\Booking\BookingsTestHelper;
+
+class BookingUpdateTest extends TestCase
+{
+    use RefreshDatabase, MakesGraphQLRequests, BookingsTestHelper;
+
+    private $position;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->actingAs($this->mockedUser());
+        $this->mockUserFind();
+
+        $this->position = factory(Position::class)->create();
+    }
+
+    /** @test */
+    public function testItCanUpdateABooking()
```

Comment:
Might want to add in a test to ensure the booking validator service is called (i.e. test it with an invalid update and check it correctly flags as invalid) Otherwise we don't test that the update has to be valid anywhere, and the change at https://github.com/VATSIM-UK/ukts-core-api/pull/33/files/9fb3cdbf2eafc4d8f40289867a400fbb03872f29..fddc2168e274335ff00dac8ef969229861efafed#diff-72faa53448f4e2c9a93c3eeafefdb65cR19 would pass either way

Reply:
added new tests in